NGCHM/WebContent/javascript/UserHelpManager.js

```diff
@@ -734,15 +734,15 @@
   //
 
   // Create a new message box.
-  UHM.newMessageBox = function (name) {
+  UHM.newMessageBox = function (name, templateId) {
```

Comment:
The comment above this change should be updated--it still says only Video Tutorial message box is implemented.